### PR TITLE
fix: market depth from order event changes

### LINKED ISSUE #7637: A GTT order that expires should cause the volume at its price to be reduced in market depth
ExpiryOrder contains the order Ids for the GTT orders which expire but market depth is not not updated correctly after expiry.

We still see volume at the GTT order price which expired

Failing test  for evidence :- test_gtt_order_that_expires

### PR DESCRIPTION
**Pull request by jeremyletang**
close #7637 .

### CODE REVIEW

**Comment by vega-paul** [comment]
```{"system-tests":"order_expiry"}```

File: cmd/data-node/commands/start/sqlsubscribers.go [hunk 1]
```diff
@@ -122,9 +122,9 @@ type SQLSubscribers struct {
 	keyRotationsSub         *sqlsubscribers.KeyRotation
 	ethereumKeyRotationsSub *sqlsubscribers.EthereumKeyRotation
 	nodeSub                 *sqlsubscribers.Node
-	marketDepthSub          *sqlsubscribers.MarketDepth
-	pupSub                  *sqlsubscribers.ProtocolUpgrade
-	snapSub                 *sqlsubscribers.SnapshotData
+	// marketDepthSub          *sqlsubscribers.MarketDepth
```
Comment (peterbarrow) [inline cmd/data-node/commands/start/sqlsubscribers.go:125]:
```suggestion
```

File: cmd/data-node/commands/start/sqlsubscribers.go [hunk 2]
```diff
@@ -162,7 +162,7 @@ func (s *SQLSubscribers) GetSQLSubscribers() []broker.SQLBrokerSubscriber {
 		s.multiSigSignerEventSub,
 		s.keyRotationsSub,
 		s.nodeSub,
-		s.marketDepthSub,
+		// s.marketDepthSub,
```
Comment (peterbarrow) [inline cmd/data-node/commands/start/sqlsubscribers.go:165]:
```suggestion
```

File: cmd/data-node/commands/start/sqlsubscribers.go [hunk 3]
```diff
@@ -298,7 +298,7 @@ func (s *SQLSubscribers) SetupSQLSubscribers() {
 	s.keyRotationsSub = sqlsubscribers.NewKeyRotation(s.keyRotationsService)
 	s.ethereumKeyRotationsSub = sqlsubscribers.NewEthereumKeyRotation(s.ethereumKeyRotationsService)
 	s.nodeSub = sqlsubscribers.NewNode(s.nodeService)
-	s.marketDepthSub = sqlsubscribers.NewMarketDepth(s.marketDepthService)
+	// s.marketDepthSub = sqlsubscribers.NewMarketDepth(s.marketDepthService)
```
Comment (peterbarrow) [inline cmd/data-node/commands/start/sqlsubscribers.go:301]:
```suggestion
```

**Review by peterbarrow — DISMISSED** [review]

**Review by EVODelavega — DISMISSED** [review]

File: cmd/data-node/commands/start/sqlsubscribers.go (hunk 1 above)
Comment (jeremyletang) [reply cmd/data-node/commands/start/sqlsubscribers.go]: addressed

File: cmd/data-node/commands/start/sqlsubscribers.go (hunk 2 above)
Comment (jeremyletang) [reply cmd/data-node/commands/start/sqlsubscribers.go]: addressed

File: cmd/data-node/commands/start/sqlsubscribers.go (hunk 3 above)
Comment (jeremyletang) [reply cmd/data-node/commands/start/sqlsubscribers.go]: addressed

**Review by EVODelavega — APPROVED** [review]

**Pull request merged**